data-prepper-plugins/mutate-event/src/test/java/com/amazon/dataprepper/plugins/processor/mutateevent/DeleteProcessorTests.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.mutateevent;
+
+import com.amazon.dataprepper.metrics.PluginMetrics;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.event.JacksonEvent;
+import com.amazon.dataprepper.model.record.Record;
+import org.junit.Test;
```

Comment:
This is a Junit 4 import. It should be Junit 5 `org.junit.jupiter.api.Test`. I believe this is in some of your other Test files as well.